src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -134,14 +134,27 @@ public function collect(Request $request, Response $response, \Exception $except
             );
         }
 
+        $this->data['voter_details'] = array();
         // collect voters and access decision manager information
         if ($this->accessDecisionManager instanceof TraceableAccessDecisionManager) {
-            $this->data['access_decision_log'] = $this->accessDecisionManager->getDecisionLog();
-            $this->data['voter_strategy'] = $this->accessDecisionManager->getStrategy();
```

Comment:
We *could* have kept it here and just use `$this->data['access_decision_log']` below to keep the diff a bit more compact.

Reply:
No problem, done.